---
layout: page
title: Days
permalink: /days/
---


# BiGre Days: February, 20th, 2025 @ IMAG

Join us for the BiGre Days, a one-day conference bringing together the vibrant
community of computational biology, biostatistics, bioinformatics, and
biophysics in Grenoble. The event will feature two invited speakers, as well
as a series of oral presentations and a poster session showcasing exciting
research.

We aim to provide a platform for young local researchers to share their work
and foster collaboration.

*The event is free but registration is mandatory*.

[Registration and call for abstracts](https://forms.gle/W74PL81FYdGJXGK4A)

## Important dates

- **Conference**: February 20th 2025
- **Announcing talk / poster selection**: Januray, 31st 2025
- **Call close**: January 15th, 2025
- **Call opening**: November, 27th, 2024

## Keynotes

We have two invited speakers scheduled!

- Claire Lema√Ætre, GenScale, Inria, Rennes
- Guido Uguzzoni, BGE, CEA, Grenoble


## Schedule

|---|---|---|
| 8:30 | 9:00 |‚òï Coffee and registration |  
|---|---|---|
| 9:00 | 9:50 | ‚≠ê Claire Lemaitre: Moving from reference genomes to pangenome graphs: benefits and challenges for the analysis of structural variation  |
| 9:50 | 10:10 | Arnaud Belcour: Predicting coarse-grained representations of biogeochemical cycles from metabarcoding data  |
| 10:10 | 10:30 | Two-dimensional numerical resolution of a polar active matter model applied to collective cell migration  |
|---|---|---|
| 10:30 | 11:00 |‚òï Coffee Break  |
|---|---|---|
| 11:00 | 11:20 | Peritumoral tissue is a promising source of prognostic biomarkers  |
| 11:20 | 11:40 | Placental DNAm changes mediating the association between prenatal air pollution exposure and child lung function: a study in the SEPAGES cohort  |
| 11:40 | 12:00 | Guilt-by-association (GBA) centrality - an interactome-based analysis method for disease gene prioritization  |
| 12:00 | 12:20 | Benchmark of microRNA targets prediction algorithms using microRNA-mRNA co-sequencing datasets at the single-cell level  |
|---|---|---|
| 12:20 | 13:30 |üçï Lunch  |
|---|---|---|
| 1:30 | 14:50 | Prediction of proteins belonging to a gene family involved in bacterial secretion systems using machine learning approaches.  |
| 1:50 | 14:10 | Unwelcome guests: characterizing the ecology and evolution of insertion sequences in prokaryotic genomes  |
| 14:10 | 14:30 | HDMAX2, an R package to perform high dimension mediation analysis  |
| 14:30 | 14:50 | Navigating Variable Selection and Mixture Identification: A Comprehensive Simulation Study with Mixture Models  |
|---|---|---|
| 14:50 | 15:40 |‚òï Coffee and poster session  |
|---|---|---|
| 15:40 | 16:00 | The study of quinone exchanges in the gut microbiota  |
| 16:00 | 16:20 | Predicting compensatory mutations in proteins using statistical models  |
| 16:20 | 17:10 |‚≠ê  Guido Uguzzoni: In-silico Design of Antibody Specificity from Phage Display Sequencing Data  |
|---|---|---|

## List of talks and posters (title and abstract)



## Organizing committee

- [Sophie Abby](https://sophieabby.github.io/), TIMC
- [Antoine Frenoy](https://perso.crans.org/frenoy/), TIMC
- [Magali Richard](https://magrichard.github.io/index.html), TIMC
- [Nelle Varoquaux](https://nellev.github.io), TIMC

## Program committee

- Thomas Burger, EDyP
- Laurent Guyon, CEA
- Clovis Galiez, LJK
- Vincent Lefort, LECA
- Fr√©d√©ric Boyer, LECA
- Delphine Ropers, INRIA, LiPHY
- Ang√©lique Stephanou, TIMC
- Ivan Junier, TIMC
- Florent Chuffart, IAB
